src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1041,6 +1042,15 @@ because xbuild doesn't support framework reference assemblies.
 	<Warning Code="XA0109" Text="Unsupported or invalid %24(TargetFrameworkVersion) value of 'v4.5'. Please update your Project Options." Condition=" '$(TargetFrameworkVersion)' == 'v4.5' "/> 
 </Target>
 
+<Target Name="_CheckSupportedAbis">
+	<PropertyGroup>
+		<AndroidSupportedAbis>;$(AndroidSupportedAbis);</AndroidSupportedAbis>
```

Comment:
This will update the existing `AndroidSupportedAbis` property group with the leading and trailing `;`. Is that the intention? If not perhaps have a local property which the Error code does the condition on. 

`<_RequestedAbis>;$(AndroidSupportedAbis);</_RequestedAbis >`

Reply:
Ah, yes! I meant to ask about that actually. I wasn't sure which way would be the preferred style. After looking at it again today, I think that since the `AndroidSupportedAbis` property is something users can edit themselves, they might indeed spend time wondering why there are extra semicolons appearing in it in the diagnostic build output. So I think it is probably worth adding a separate property as you suggest, at the small cost of a little extra "noise" in the list of property names used during the build. I will update the pull request in a moment.

Another option if it might be OK to be slightly less precise about the string matching would be to skip adding any semicolons and instead adjust the error `Condition` to:

    Condition="$(AndroidSupportedAbis.Contains('armeabi;')) Or $(AndroidSupportedAbis.EndsWith('armeabi'))"

I'd be happy to update this pull request again if that sounds better. Thanks!